website/tutorial/interacting-with-a-database-server.md

```diff
@@ -0,0 +1,71 @@
+---
+sidebar_position: 5
+title: Interacting with a database server
+description: Interacting with databases from the Podman Desktop application
+keywords: [podman desktop, podman, databases]
+tags: [podman-desktop, interacting-with-databases]
+---
+
+# Interacting with a database server
+
+This tutorial covers the following end-to-end tasks required to interact with a database server from within the Podman Desktop UI:
+
+- Pulling a SQL server image
+- Creating a SQL server instance
+- Accessing the instance from terminal
+
+For creating a database instance, you can use one of the following options:
+
+- [Build a MySQL server image using a container or docker file](/docs/containers/images/building-an-image)
+- [Import a MySQL server image from your local machine](/tutorial/managing-your-application-resources#managing-images) using the **Import** button on the _Images_ component page
+- Pull a MySQL server image from a registry (covered in this tutorial)
+
+This tutorial focuses on creating a MySQL server instance by pulling a MySQL server image from the _quay.io_ registry.
+
+## Before you begin
+
+Make sure you have:
+
+- [Installed Podman Desktop application](/docs/installation).
+- [A running Podman machine](/docs/podman/creating-a-podman-machine).
+- [Configured the _quay.io_ registry](/docs/containers/registries) to pull a MySQL Server image, such as `quay.io/fedora/mysql-80`.
+- A developer role.
```

Comment:
I thought that a user requires to configure this registry before pulling an image --- as a prerequisite. But, this is not the case. So, i removed the bullet point.

Reply:
I was thinking the "developer role" (I should specify better), I would imagine that the developer role has something to do with the quay.io, but since it is now removed, I think that this should be removed too

Reply:
I see the point of concern. @gastoner, the complete prerequisite reads as `Make sure you have a developer role.`  this is basically clarifying for whom (audience) this content is for or who is the audience for this content? Should i add Database Administrators as well??